tests/_internal/test_client.py

```diff
@@ -262,6 +332,33 @@ async def test_async_client_list_clients_can_handle_empty_response():
     assert services == []
 
 
+@respx.mock
+def test_client_list_links_can_handle_empty_response():
+    # Given
+    respx.get("https://test.example.com/v1/links").respond(json={})
+    cl = client.Client("https://test.example.com")
+
+    # When
+    services = cl.list_links()
+
+    # Then
+    assert services == []
```

Comment:
jfyi/nit: `services` -> `links`.  same for following test.

Reply:
THX. Fixed!